### PR TITLE
release notes: v12 preview

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/v12.0.0.md [hunk 1]
```diff
@@ -13,9 +13,245 @@ This page is work in progress, come back soon for updates!
...
 
 ## Fast Data
 
+### Fast Data Configurator
+
+#### Unique values for ingestion topics and pr-update topics
+
+To further improve the automatic configuration of the Fast Data, from now on when configuring a Projection inside a System of Record it is possible to set only one _ingestion topic_ and only one _pr-update topic_, instead of a topic for each environment.
+
+Every new projection created will automatically set the ingestion topic to the pattern `{tenantId}.{projectId}.{systemId}.{projectionId}.ingestion` and the pr-update topic with `{tenantId}.{projectId}.{systemId}.{projectionId}.pr-update`. These are editable and it is possible to use environment variables for the entire name or part of the name.
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:46]:
```suggestion
Every new projection created will automatically set the ingestion topic to the pattern `{tenantId}.{projectId}.{systemId}.{projectionId}.ingestion` and the pr-update topic with `{tenantId}.{projectId}.{systemId}.{projectionId}.pr-update`. These are editable and it is possible to use environment variables for the entire name or part of the name. This lets you differentiate topics for each environment leveraging on the usage of environment variables.
```

File: docs/release-notes/v12.0.0.md [hunk 2]
```diff
@@ -13,9 +13,245 @@ This page is work in progress, come back soon for updates!
...
+![Single View Trigger Generator page](../fast_data/configuration/img/svtg-configuration.png)
+
+Attaching a Single View Trigger Generator will simplify the configuration of the service itself by automatically setting the same ER Schema config map used in the Single View Creator, and including a dedicated space where to configure the [Kafka Projection Updates](/fast_data/configuration/config_maps/kafka_projection_updates.md) and the [Projection Changes Schema](/fast_data/configuration/config_maps/projection_changes_schema.md) config map.
+
+:::info
+This configuration cannot be used in case the Kafka Projection Updates uses manual strategies
+:::
+
+#### Upsert/Delete Strategy configuration page
+feat:
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:77]:
```suggestion
```

File: docs/release-notes/v12.0.0.md [hunk 3]
```diff
@@ -13,9 +13,245 @@ This page is work in progress, come back soon for updates!
...
+Put here the description of the feature. Remember to add the documentation link
+
+![add here your picture if needed and give it a name](./img/path/to/your/picture.png)
+
+#### Improvements
+
+##### Put here the title of the improvement
+
+Put here the description of the improvement
+
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:97]:
```suggestion
```

File: docs/release-notes/v12.0.0.md [hunk 4]
```diff
@@ -13,9 +13,217 @@ This page is work in progress, come back soon for updates!
...
+This version addressed few bugs, here a comprehensive list:
+
+* list here all the bug fix
+* but do not highlight them so much
+* just put a short description 
+
+## Back-Kit Library
+
+The new version `v1.4.7` of the Backoffice library (back-kit) is available!
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:160]:
```suggestion
The new version `v1.4.7` of the Back-kit library is available!
```

File: docs/release-notes/v12.0.0.md [hunk 5]
```diff
@@ -13,9 +13,217 @@ This page is work in progress, come back soon for updates!
...
+
+This version addressed few bugs, here a comprehensive list:
+
+* list here all the bug fix
+* but do not highlight them so much
+* just put a short description 
+
+For more information check out the service changelog (add link to the changelog)
+
+## Microfrontend Composer
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:134]:
```suggestion
## Backoffice becomes Microfrontend Composer

:::info
Backoffice has a brand new name: Microfrontend Composer! From now on, for all the features related to the formerly known Backoffice Configurator, you will find references to Microfrontend Composer. The [documentation](/microfrontend-composer/what-is.md) has been updated accordingly.
:::
```

File: docs/release-notes/v12.0.0.md [hunk 6]
```diff
@@ -13,9 +13,231 @@ This page is work in progress, come back soon for updates!
...
+:::caution
+At the moment, Listeners are only available for Projects using an **Envoy API Gateway**.
+:::
 
 ## Fast Data
 
+### Fast Data Configurator
+
+#### Import and Export of Fast Data resources between projects
+
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:43]: @fedepini aggiungiamo un link alla doc?

Comment (fedepini) [reply docs/release-notes/v12.0.0.md:43]: @TheRusCompass fatto

File: docs/release-notes/v12.0.0.md [hunk 7]
```diff
@@ -13,9 +13,251 @@ This page is work in progress, come back soon for updates!
...
+### Fast Data Services
+
+#### Projection Storer
+
+The version `v1.0.0` of the _Projection Store_ is now available!
+
+This is a new service that can be employed in Fast Data _event-driven_ architecture in place of
+Real-Time Updater plugin.  
+Similarly to Real-Time Updater plugin, Projection Storer consumes change events as ingestion messages,
+transform them into projection records, filtering and validating each of their fields and eventually save them onto the specified storage system, emitting a proper notification for downstream components.
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:119]:
```suggestion
Similarly to Real-Time Updater plugin, Projection Storer consumes change events as ingestion messages,
transforms them into projection records, filters and validates each of their fields and eventually saves them onto the specified storage system, emitting a proper notification for downstream components.
```

File: docs/release-notes/v12.0.0.md [hunk 8]
```diff
@@ -13,9 +13,251 @@ This page is work in progress, come back soon for updates!
...
+
+###### Added support for side effects fired alongside commands
+
+For each state of the machine it is now possible to specify an array of side effects command to file alongside the main command in a fire-and-forget manner.
+
+To learn more about this feature, check out the dedicated [documentation page](/runtime_suite/flow-manager-service/30_configuration.md#states-of-the-machine)
+
+###### Added hook functions to build commands and side effects custom payload
+
+Commands payload can now be generated thought a custom function that takes as input the entire Saga entity and the message label of the command, and returns as output an object with the payload to use.
```
Comment (TheRusCompass) [inline docs/release-notes/v12.0.0.md:251]:
```suggestion
Commands payload can now be generated through a custom function that takes as input the entire Saga entity and the message label of the command, and returns as output an object with the payload to use.
```

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**